src/v0/destinations/braze/util.js

```diff
@@ -634,6 +634,10 @@ function getPurchaseObjs(message) {
       parseInt(quantity, 10),
       timestamp,
     );
+    const extraProperties = _.omit(product, ['product_id', 'sku', 'price', 'quantity', 'currency']);
```

Comment:
should be enable this via destination config to avoid impacting all customer, we don't know if everyone needs this or not

Reply:
Its not a good `product UX` that for a small feature like this we add a config in UI?

Reply:
We are handling this behind a destination config